examples/storm-elasticsearch-examples/src/main/java/org/apache/storm/elasticsearch/common/EsConstants.java

```diff
@@ -15,8 +15,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.storm.elasticsearch.common;
 
-public class EsConstants {
-    public static String clusterName = "test-cluster";
+/**
+ * Constants used in ElasticSearch examples.
+ * @author richter
+ */
+public final class EsConstants {
+
+    /**
+     * The cluster name.
+     */
+    public static final String CLUSTER_NAME = "test-cluster";
+    /**
+     * The default wait value in seconds. Necessary in order to avoid magic
+     * number anti-pattern.
+     */
+    public static final int WAIT_DEFAULT = 5;
```

Comment:
Nit: Putting this in a field is a good change. Could you rename the field so it contains the unit (e.g. WAIT_DEFAULT_SECS)?